io/swig/io/mechanics_hdf5.py

```diff
@@ -929,6 +930,8 @@ def add_convex_shape(self, name, points,
             shape[:] = points[:]
             shape.attrs['type'] = 'convex'
             shape.attrs['id'] = self._number_of_shapes
+            if avoid_internal_edge_contact:
+                shape.attrs['avoid_internal_edge_contact'] = True
```

Comment:
A detail, this can be written ```shape.attrs['avoid_internal_edge_contact'] = avoid_internal_edge_contact:```

Reply:
Initially, I wanted to have the possibility to test if the attributes are existing or not.